stripe.go

```diff
@@ -977,6 +977,10 @@ func ParseID(data []byte) (string, bool) {
 		return "", false
 	}
 
+	if s == "\"" {
```

Comment:
Mind also porting the comment from below up here? Maybe something like this:

> Edge case that should never happen; found via fuzzing

Reply:
This totally makes sense. I've added a comment here and updated a comment in a test.